java_src/foam/dao/EnabledAwareDAO.java

```diff
@@ -14,13 +14,13 @@
 
 public class EnabledAwareDAO
   extends ProxyDAO
-  implements EnabledAware
 {
+  private Predicate PREDICATE_ = EQ(this.EnabledAware.ENABLED, true);
```

Comment:
Should be 'final' and 'static' to be a constant. Otherwise you're still creating multiple instants and users could update the value.